README.md

```diff
@@ -1,12 +1,15 @@
+<!--$UNCOMMENT(ray-lightning)=-->
+
 # Distributed PyTorch Lightning Training on Ray
 This library adds new PyTorch Lightning plugins for distributed training using the Ray distributed computing framework.
 
 These PyTorch Lightning Plugins on Ray enable quick and easy parallel training while still leveraging all the benefits of PyTorch Lightning and using your desired training protocol, either [PyTorch Distributed Data Parallel](https://pytorch.org/tutorials/intermediate/ddp_tutorial.html) or [Horovod](https://github.com/horovod/horovod). 
 
 Once you add your plugin to the PyTorch Lightning Trainer, you can parallelize training to all the cores in your laptop, or across a massive multi-node, multi-GPU cluster with no additional code changes.
 
-This library also comes with an integration with [Ray Tune](tune.io) for distributed hyperparameter tuning experiments.
+This library also comes with an integration with [Ray Tune](https://docs.ray.io/en/latest/tune/index.html) for distributed hyperparameter tuning experiments.
```

Comment:
```suggestion
This library also comes with an integration with <!--$UNCOMMENT{ref}`Ray Tune <tune-main>`--><!--$REMOVE-->[Ray Tune](https://tune.io)<!--$END_REMOVE--> for distributed hyperparameter tuning experiments.
```

Reply:
tune.io now just redirects to the docs page anyways-- I think we got rid of the website itself.